xc7/tests/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-add_subdirectory(buttons)
+#add_subdirectory(buttons)
 add_subdirectory(counter)
-add_subdirectory(ram_test)
-add_subdirectory(carry)
-add_subdirectory(simple_ff)
-add_subdirectory(dram)
-add_subdirectory(bram)
+#add_subdirectory(ram_test)
+#add_subdirectory(carry)
+#add_subdirectory(simple_ff)
+#add_subdirectory(dram)
```

Comment:
Why comment these out?  If you don't want to build them, just don't run their targets?